gate-saml/gate-saml.gradle

```diff
@@ -0,0 +1,19 @@
+repositories {
+  maven {
+    url 'https://repo.spring.io/libs-snapshot'
+  }
+}
+
+dependencies{
+  compile project(':gate-core')
+  // RetrySupport is in kork-exceptions and not kork-core!
+  compile spinnaker.dependency("korkExceptions")
+
+  compile('org.opensaml:opensaml:2.6.4')
+  compile("org.springframework.security.extensions:spring-security-saml2-core:1.0.2.RELEASE")
+  compile("org.springframework.security.extensions:spring-security-saml-dsl:1.0.0.M3") {
```

Comment:
It appears that this was officially released, so you can use the `1.0.5.RELEASE` and remove the `repositories` block above.

Reply:
Ack. Will have to do this in a later PR since updating to `1.0.5.RELEASE` does not seem to just work because `org.springframework.security.extensions.saml2.config.SAMLConfigurer` cannot be resolved.